utils/http.go

```diff
@@ -56,14 +59,13 @@ func (root *rootHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	ctx, w = ctxu.WithResponseWriter(ctx, w)
 	ctx = ctxu.WithLogger(ctx, log)
 	ctx = context.WithValue(ctx, notary.CtxKeyCryptoSvc, root.trust)
-
-	defer func() {
+	defer func(ctx context.Context) {
 		ctxu.GetResponseLogger(ctx).Info("response completed")
-	}()
+	}(ctx)
```

Comment:
`doAuth` does not currently updates the logger, but I suppose it's possible in the future that it may.  I'm not sure if we necessarily care though about all the other fields in a "response completed" log.